modular_skyrat/modules/cargo/code/packs.dm

```diff
@@ -92,7 +92,7 @@
 /datum/supply_pack/medical/medigun
 	name = "CWM-479 Medigun"
 	desc = "Contains a single VeyMedical CWM-479 model medical gun; cells not included."
-	cost = CARGO_CRATE_VALUE * 30
+	cost = CARGO_CRATE_VALUE * 100
```

Comment:
Do you mean to have them cost 20k credits?

Reply:
> Do you mean to have them cost 20k credits?

isn't that only 2k?

Reply:
Wildcats which are crate value * 58 range from 11k-16k depending on a few things

* 100 could easily hit 20k credits

Reply:
On this note, this nerf can ONLY work in conjunction with moving it to the misc supply category. Medicells CANNOT be increased in price and remain orderable via medical supply consoles, because that's an insane economy oversight. For reference, Industrial RCD's cost Crate value * 40, which is the general upper limit for crate value costs apart from the Wildcats

Personally this nerf is something I disagree with, however if you are doing this, you simply cannot leave it in the medical supply console list.
And if you're doing that, then medical will just. Never be able to buy them for this price.

👎

Reply:
>isn't that only 2k?
The medigun right now costs 6k, this would increase it to 20k